MLS.Agent/Program.cs

```diff
@@ -139,9 +143,16 @@ public static IWebHost ConstructWebHost(StartupOptions options)
                           })
                           .UseEnvironment(options.EnvironmentName)
                           .UseStartup<Startup>()
+                          .UseUrls(uri.ToString())
                           .Build();
 
             return webHost;
         }
+
+        private static bool IsInToolMode()
```

Comment:
I think this should be called something like `IsLaunchedForDevelopment` or something like that. Basically this will be true when someone launches `dotnet-try` from source in Visual Studio, Visual Studio Code, or `dotnet run`.